pkg/printer/printer.go

```diff
@@ -0,0 +1,74 @@
+package printer
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+)
+
+type Config struct {
+	Err io.Writer
+	Out io.Writer
+}
+
+type Printer struct {
+	*decoratedPrinter
+
+	err io.Writer
+	out io.Writer
+}
+
```

Comment:
What do you think about placing it here?

```suggestion

var _ Interface = &Printer{}

```